mypy_baseline/commands/_filter.py

```diff
@@ -37,12 +37,13 @@ def run(self) -> int:
                 unresolved_errors.append(error)
 
         fixed_errors: list[Error] = []
-        for line in baseline:
-            error = Error.new(line)
-            if error is None:
-                print(f'invalid baseline, cannot parse line: {line}')
-                return 1
-            fixed_errors.append(error)
+        if baseline != [""]: # When there are no errors, baseline is empty
```

Comment:
What is we just skip empty lines? It will make the baseline parsing more resilient. So, instead of this check, check in the loop:

```python
if not line:
    continue
```

The reason is that `!= [""]` check feels oddly specific. For example, a `\n` file won't pass but empty file would, which might lead to unexpected results.

Reply:
Fixed